docs/openapi_v1.0.0-alpha.yaml

```
@@ -875,21 +920,23 @@ components:
             - CANCELLED
           description: |
             After a successful `createReservation` or `createCancellation` request, the `status` MUST be `ON_HOLD`.
-            
+
             After a successful `confirmReservation` request, the `status` MUST be `CONFIRMED`.
-            
+
             After a successful `confirmCancellation` request, the `status` MUST be `CANCELLED`.
-            
+
             Following are the only valid Reservation flow status transitions:
-            
+
             * New Reservation -> `REJECTED`
             * New Reservation -> `ON_HOLD` -> `EXPIRED`
             * New Reservation -> `ON_HOLD` -> `CONFIRMED`
             * New Reservation -> `ON_HOLD` -> `PENDING` -> `REJECTED`
             * New Reservation -> `ON_HOLD` -> `PENDING` -> `CONFIRMED`
-            
+
+            The `PENDING` status MAY appear only for products with the `instantConfirmation` property set to `false`. Poll the /bookings endpoint for status changes.
```

I believe I understand the intention you are trying to convey and would suggest the following tweaks to help remove any confusion. It took me a few reads to make sure I was parsing it the way I believe you meant it. Please confirm if I've understood your intention and clarified.

```suggestion
            The `PENDING` status MAY NOT be returned if the Product definition shows `"instantConfirmation": true`. When the Product definition shows `"instantConfirmation": false` a `PENDING` status SHOULD be returned until the Reservation has been confirmed by the Supplier. When receiving a status of `PENDING`, the Reseller SHOULD poll the /bookings endpoint for status changes to know when they may move to the next step in the workflow.
```

---

Applied your suggestion